### DISCUSSION TITLE
Do helicopter events happen off screen?

### DISCUSSION THREAD

**Discussion by Toby4612** [TEHE-Studios/ExpandedHelicopterEvents#252] [Q&A] [answered]
Say I'm in rosewood, could a FEMA drop or something else similar happen in some other map area like Louisville? just curious.

**Comment by Chuckleberry-Finn** [comment, marked as answer]
Yes, but events usually aim for areas of action. Players contribute to a "heatMap" that tracks certain actions - think of it as drawing attention.
